src/components/ButtonsLanguages.jsx

```diff
@@ -1,19 +1,13 @@
 import style from './ButtonLanguage.module.css';
 import { useCharacteristicsContext } from '../hooks/useCharacteristicsContext';
 
-function ButtonLanguage({ children, close }) {
+function ButtonLanguage({ children, parentName }) {
   const { handleSelect, handleRemove } = useCharacteristicsContext();
   return (
-    <>
-      <p onClick={() => handleSelect(children)} className={style.btn}>
-        {children}
-        {close ? (
-          <button onClick={() => handleRemove(children)} className={style.close}></button>
-        ) : (
-          ''
-        )}
-      </p>
-    </>
+    <button onClick={() => handleSelect(children)} className={style.btn}>
+      {children}
+      {parentName ? <a onClick={() => handleRemove(children)} className={style.close}></a> : ''}
```

Comment:
What `parentName` means in this context?

And why did you choose `a` tag inside of a button instead a regular button?

Reply:
I used it to know which element this button is from. This allows you to add a cross(remove) to it

Reply:
'a' I used beacouse prohibited to use button in button

Reply:
I don't know how to understand it why it's ugly and bad.

Look, when you wanna use it to remove a filter you still can click on a whole button, so you don't need a second one inside. All you need is a different handler and a boolean flag saying "add x sign".

Reply:
I can’t understand how we will change this flag?

Reply:
You can use a boolean flag `close` like you did before or add a property variant with a string values. 

One component will always use version with close icon and another - always without it.

```
// First option
<ButtonsLanguages onClick={addToFilters} />

<ButtonsLanguages onClick={removeFromFilters} isClose />


// Second option
<ButtonsLanguages onClick={addToFilters} variant="add" />

<ButtonsLanguages onClick={removeFromFilters} variant="remove" />
```